shopinvader/services/abstract_download.py

```diff
@@ -69,7 +69,7 @@ def _get_binary_content(self, target, params=None):
         target_report_def = self._get_report_action(target, params=params)
         report_name = target_report_def.get("report_name")
         report_type = target_report_def.get("report_type")
-        report = self._get_report(report_name, report_type)
+        report = self.env["ir.actions.report"]._get_report_from_name(report_name)
```

Comment:
what about `_get_report` then? Remove? Deprecate? Fix?

Reply:
Mmmh, yes indeed. the good approach should have been modifying that method instead no ?